srcpkgs/accountsservice/template

```diff
@@ -24,20 +21,19 @@ build_options="gir"
 # Disable gir for cross builds.
 if [ -z "$CROSS_BUILD" ]; then
 	build_options_default="gir"
+else
+	# for ITS rules
+	hostmakedepends+=" polkit-devel"
 fi
 
-pre_configure() {
-	case "$XBPS_TARGET_MACHINE" in
-		*-musl)
-			# Patch to check for /var/log/wtmp instead of /var/log/utx.log
-			sed -i configure -e "s;utx\.log;wtmp;g"
-			configure_args+=" ac_cv_file__var_log_utx_log=yes"
-			# FIXME: if ac_cv_header_utmpx_h=no is set, the function
-			# wtmp_helper_update_login_frequencies() is not defined.
-			# configure_args+=" ac_cv_header_utmpx_h=no"
-			;;
-	esac
-}
+case "$XBPS_TARGET_MACHINE" in
+	*-musl)
+		pre_configure() {
+			patch -Np0 -i "${FILESDIR}"/musl-wtmp.patch
```

Comment:
Does it really need to be only on musl ?

Reply:
Ah, no, glibc shouldn't even reach the `elif` statement the patch changes